tasks/notify.py

```diff
@@ -399,6 +402,55 @@ def run_impl_within_lock(
             )
             return {"notified": False, "notifications": None}
 
+    def _possibly_refresh_previous_selection(self, commit: Commit) -> bool:
```

Comment:
I don't want to change this because (1) I don't feel "maybe" is an improvement to "possibly" and (2, most important) we use "possibly_" in this context in other parts of the code, so it keeps consistent

Reply:
On second thought, `maybe` doesn't necessarily fit here because it's more commonly used for Optional types (see [monads](https://en.wikipedia.org/wiki/Monad_(functional_programming)#An_example:_Maybe)). So I'm fine leaving it.